### PR TITLE
modify rnn/lstm

### PR DESCRIPTION
**Pull request by MESeraph**

### CODE REVIEW

File: src/tim/vx/internal/include/ops/vsi_nn_op_bidirectional_sequence_lstm.h [hunk 1]
```diff
@@ -48,68 +48,83 @@ enum
...
+    BI_LSTM_FW_INPUT_BIAS_I2F         = 13,
+    BI_LSTM_FW_INPUT_BIAS_I2C         = 14,
+    BI_LSTM_FW_INPUT_BIAS_I2O         = 15,
+
+    BI_LSTM_FW_INPUT_BIAS_R2I        = 16,
+    BI_LSTM_FW_INPUT_BIAS_R2F         = 17,
+    BI_LSTM_FW_INPUT_BIAS_R2C         = 18,
+    BI_LSTM_FW_INPUT_BIAS_R2O         = 19,
+
+    BI_LSTM_FW_INPUT_WEIGHT_PROJ    = 20,
```
Comment (sunshinemyson) [inline src/tim/vx/internal/include/ops/vsi_nn_op_bidirectional_sequence_lstm.h:61]: Why you need to change the order?

**Comment by sunshinemyson** [comment]
@MESeraph This will broken our tflite delegate implementation.

Could you explain more about the background?

**Comment by MESeraph** [comment]
@sunshinemyson 
I just add recurrence bias. Need the tesnor in ONNX.
Screen Shot 2022-12-12 at 10 35 35 AM

**Comment by sunshinemyson** [comment]
Could you append the new operand to tail? If so, we don't have to rewrite all the test/mapper.

Thanks

**Comment by MESeraph** [comment]
@sunshinemyson ok

**Pull request closed without merging**